src/solvers/dgsem_tree/indicators.jl

```diff
@@ -101,6 +101,78 @@ function Base.show(io::IO, ::MIME"text/plain", indicator::IndicatorHennemannGass
   end
 end
 
+"""
```

Comment:
Please move the new stuff a bit below so that the standard version comes first when reading the file from top to bottom

Reply:
Do you want the code to be right before `IndicatorLöhner` and after `function (indicator_hg::IndicatorHennemannGassner)` ?